server/models/visits.js

```diff
@@ -83,6 +83,7 @@ var visits = {
                 fields: [
                   'extractedTitle',
                   'extractedContent',
+                  'extractedDescription',
```

Comment:
Unrelated, but should we also be searching `extractedUrl`, `extractedProviderUrl`, and/or `title`?

Basically, "title" because the `extractedTitle` will be less useful if the scraped page was behind a login (ie: facebook, or something on localhost that embedly can't touch).